src/components/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import UserProfile from './UserProfile';
+import './Styles/Navbar.css';
+
+const Navbar = () => (
+  <header className="header">
+    <div className="navbar">
+      <h1>Bookstore CMS</h1>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Books</Link>
+          </li>
+          <li className="active">
+            <Link to="categories">Categories</Link>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <UserProfile />
+
+  </header>
```

Comment:
- [ ] Good job so far, :clap: , however, please ensure that your styles match the given design, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. Please resolve it. :slightly_smiling_face: 
- [ ] The color code for the book font on the Nav is `#121212` 
- [ ] Please remove the border color on your profile icon and leave it just as is in the given design.
- [ ] The color code of your circle is a gradient color and all of this is given in the design please resolve it.
- [ ] And please remove the hover color on your add button. :slightly_smiling_face: 

Below is an illustration of yours and the given Design please pay on red markings  :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Last](https://github.com/efmomoh/bookstore/assets/43528594/12156528-4213-4483-b5b8-6b94eecf5157)

</td>
      <td>

![eplin Design BookStore](https://github.com/lucy-sees/React-Bookstore-App/assets/43528594/392e5194-eb28-4316-a9bc-6e67c2885962)

</td>
    </tr>
  </table>